react/week3/src/TodoForm.js

```diff
@@ -0,0 +1,23 @@
+import React, {useState} from 'react';
+
+const TodoForm = (props) => {
+  
+  const [checked, setChecked] = useState(false)
+  const handleClick = () => setChecked(!checked)
+
+  return (
+    <li style = {{ textDecoration: checked ? "line-through" : "" , border : "1px solid black", padding:"5px", width:"500px"}}>
```

Comment:
alos here don't use inline styles,
so you will need 2 classes for this code 
you can install classnames from npm => https://jedwatson.github.io/classnames/
and do something like this instead
className={{ checked &&  "addNewClassName", "yourDefaultClassName" }}
"yourDefaultClassName" in you css file will do the default styles 
and "addNewClassName" you will add in and do the styles when checked is true

Reply:
Done. Thank you